src/main/views/suspend-user.njk

```diff
@@ -0,0 +1,60 @@
+{% from "macros/breadcrumb.njk" import breadcrumb %}
+ {% from "macros/csrf.njk" import csrfProtection %}
```

Comment:
indentation issue

Reply:
Resolved in https://github.com/hmcts/idam-user-dashboard/pull/238/commits/dcd4ffded02acb65dd55dc7a391e28ad8cb4770d